Datadog.Serverless/CompatibilityLayer.cs

```diff
@@ -169,9 +184,16 @@ public static void Start()
 
             try
             {
+                string tempDir = Path.Combine(Path.GetTempPath(), "datadog");
+                Directory.CreateDirectory(tempDir);
+                string executableFilePath = Path.Combine(tempDir, Path.GetFileName(binaryPath));
+                File.Copy(binaryPath, executableFilePath, overwrite: true);
+                SetFilePermissions(executableFilePath);
+                _logger.LogDebug("Spawning process from binary at path {binaryPath}", binaryPath);
```

Comment:
Should the log use `executableFilePath` instead to show where the file was copied into? Would that help troubleshooting?